6.2.5: Cannot find module 'import-sort-style-eslint'

I just upgraded from 6.2.4 to 6.2.5 and the extension no longer works. In the extension output I see this error:

```
Cannot find module 'import-sort-style-eslint'
```

--- a/src/config-cache.ts
+++ b/src/config-cache.ts
@@ -60,19 +60,14 @@ export function getConfig(document: TextDocument): IResolvedConfig {
   const config = clone(DEFAULT_CONFIGS);
   const defaultSortStyle = getConfiguration<string>('default-sort-style');
 
+  Object.keys(config).forEach((key) => {
+    config[key].style = `import-sort-style-${defaultSortStyle}`;
+  });
+
   // Initialize reference to the document current workspace folder
   if (!currentWorkspaceFolder) {
     currentWorkspaceFolder = workspace.getWorkspaceFolder(document.uri);
   }
-
-  Object.keys(config).forEach((key) => {
-    config[key].style = require.resolve(
-      `import-sort-style-${defaultSortStyle}`,
-      {
-        paths: [currentWorkspaceFolder.uri.path]
-      }
-    );
-  });
 
   if (!useCache || !cachedConfig || hasWorkspaceFolderChanged(document)) {
     cachedConfig = sortGetConfig(

